### ISSUE TITLE
Exif lens metadata missing when converting Pentax K-3 DNG to TIFF

### ISSUE THREAD

**Issue by vnedimovic** [Beep6581/RawTherapee#3448]
After processing DNG image to TIFF format Lens metadata is lost on Quick Info (Pentax K-3 dslr and Pentax lenses)

**Comment by heckflosse** [comment]
Please upload a raw file to filebin.net and post the link here

**Comment by vnedimovic** [comment]
https://filebin.net/hewitjrsdiy1gz0t

Filebin :: bin hewitjrsdiy1gz0t<https://filebin.net/hewitjrsdiy1gz0t>
filebin.net
Upload files and make them available for your friends. Think of it as Pastebin for files. Registration is not required. Large files are supported.




Vojin Nedimović


________________________________

**Comment by Beep6581** [comment]
ExifTool on the DNG
```-MakerNotes:LensType=HD PENTAX-DA 55-300mm F4-5.8 ED WR
-MakerNotes:LensType=HD PENTAX-DA 55-300mm F4-5.8 ED WR
-MakerNotes:LensFStops=8.5
-MakerNotes:LensKind=0x28
-MakerNotes:LensFocalLength=300.0 mm
-Composite:LensID=HD PENTAX-DA 55-300mm F4-5.8 ED WR
```

**Comment by Beep6581** [comment]
Link expired, hopefully fixed by #3802, closing.
